src/mmw/js/src/modeling/models.js

```diff
@@ -521,12 +523,13 @@ var ProjectModel = Backbone.Model.extend({
         var self = this,
             saveProjectAndScenarios = _.bind(self.saveProjectAndScenarios, self);
 
-        if (self.get('gis_data') === null && self.fetchGisDataPromise === undefined) {
+        if (self.get('mapshed_job_uuid') === null && self.fetchGisDataPromise === undefined) {
```

Comment:
Nice. This ensures that when existing projects are reloaded, they are converted to the new format.